distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -1,21 +1,21 @@
 priority "grant";
 // Double forward slashes "//" indicate comments and have no effect on policies.
-
-// This policy file's intended usage is to contain appropriate sections for adding permissions related
-// to configurations. For example, the Content Directory Monitor requires read/write permissions to
-// the directory being monitored, and has an appropriate section below. Sections should contain
-// sufficient details about adding permissions required for the given configuration.
-
-// Instructions in this file may be duplicated in the documentation. Documentation provides better
-// formatting and readability.
-
-// When adding permissions for directories, trailing slashes have no effect on the permissions granted.
-// For example, adding a permission for "${/}test${/}path" and "${/}test${/}path${/}" are equivalent. The recursive forms
-// "${/}test${/}path${/}-", and "${/}test${/}path${/}${/}-" are also equivalent.
-
+//
+// This policy file's intended usage is to contain appropriate sections for adding permissions
+// related to configurations. For example, the Content Directory Monitor requires read/write
+// permissions to the directory being monitored, and has an appropriate section below. Sections
+// should contain sufficient details about adding permissions required for the given configuration.
+//
+// Instructions in this file should be duplicated in the Documentation. The Documentation provides
+// better formatting and readability and also prevents these directions from being modified.
+//
+// When adding permissions for directories, trailing slashes have no effect on the permissions
+// granted. For example, adding a permission for "${/}test${/}path" and "${/}test${/}path${/}" are
+// equivalent. The recursive forms "${/}test${/}path${/}-" and "${/}test${/}path${/}${/}-" are also
```

Comment:
✏️ I realize this was here before this PR, but in `The recursive forms "${/}test${/}path${/}-" and "${/}test${/}path${/}${/}-"` the second example is incorrect. I think it's supposed to be `"${/}test${/}path${/}-${/}"` (the dash in between the two final slashes rather than at the very end).